packages/graphback-cli/src/init/starterTemplates.ts

```diff
@@ -50,6 +50,15 @@ export const allTemplates: Template[] = [
       path: '/templates/ts-apollo-mongodb-backend',
     }]
   },
+  {
+    name: 'apollo-mongo-datasync-server-ts',
+    description: 'Apollo GraphQL Server connecting to Mongo database using TypeScript. Contains Data Synchronization features.',
+    repos: [{
+      uri: 'https://github.com/aerogear/graphback',
+      branch: 'templates-0.14.0',
+      path: '/templates/ts-apollo-mongodb-datasync-backend',
+    }]
+  },
```

Comment:
Is this okay?

Reply:
Also a fullstack entry will be required as in https://github.com/aerogear/graphback/blob/master/packages/graphback-cli/src/init/starterTemplates.ts#L29

Reply:
Was kinda deciding against this as full-stack equivalent would be with offix perhaps and do not want to add that here :smile: 
Does client plugin produce sync queries?

Reply:
Ah good question and remark. I do not think it does so maybe let's keep it as you were suggesting. 

And this way, maybe we can remove the [client sysmink](https://github.com/aerogear/graphback/blob/29a82d5f6f66111c5d56732309f76414e1da6e0a/templates/ts-apollo-mongodb-datasync-backend/client) ?

Reply:
ah yes, definitely :smile: , did not see that one , on it